packages/playwright-core/src/server/dispatchers/electronDispatcher.ts

```diff
@@ -67,6 +79,15 @@ export class ElectronApplicationDispatcher extends Dispatcher<ElectronApplicatio
     return { handle: ElementHandleDispatcher.fromJSHandle(this, result) };
   }
 
+  async updateSubscription(params: channels.ElectronApplicationUpdateSubscriptionParams): Promise<void> {
+    if (params.enabled)
+      this._subscriptions.add(params.event);
+    else
+      this._subscriptions.delete(params.event);
+    if (params.event === 'console' && params.enabled)
+      this._object._emitBufferedConsoleMessages();
```

Comment:
This does not fit my understanding of event listeners. Usually, you do not get past events upon subscribing to an event listener. Are we trying to solve some particular usecase?